integrations/to.etc.domui.selenium/src/main/java/to/etc/domui/webdriver/core/base/SelectPO.java

```diff
@@ -0,0 +1,27 @@
+package to.etc.domui.webdriver.core.base;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openqa.selenium.By;
+import to.etc.domui.webdriver.core.WebDriverConnector;
+
+@NonNullByDefault
+public class SelectPO extends BasePO {
```

Comment:
Should not really exist by itself? I.e. we would need ComboLookup2PO and ilk which could use this as a base/part, of course..